src/nets/e2e_asr_attctc.py

```diff
@@ -412,13 +412,14 @@ def __init__(self, eprojs, odim, dlayers, dunits, sos, eos, att, verbose=0, char
         super(Decoder, self).__init__()
         with self.init_scope():
             self.embed = L.EmbedID(odim, dunits)
-            # TODO(watanabe) use multiple layers with dlayers option
-            # 310s per 100 ite -> 240s from NStepLSTM
-            self.decoder = L.StatelessLSTM(dunits + eprojs, dunits)
+            self.lstm0 = L.StatelessLSTM(dunits + eprojs, dunits)
+            for l in six.moves.range(1, dlayers):
+                self.add_link('lstm%d' % l, L.StatelessLSTM(dunits, dunits))
```

Comment:
I think it is better to use setattr(self, 'lstm%d' % l, L.StatelessLSTM(dunits, dunits)) .
add_link is deprecated as seen in https://github.com/chainer/chainer/blob/7dbf31f8cea8e7c7229a41bd7ee3dd1896a2e71e/chainer/link.py#L708

Reply:
Great suggestions. We have used add_link in the other parts. @kan-bayashi can you also change these parts to `setattr()`?

Reply:
OK! I replace `add_link` with yours.